### DISCUSSION TITLE
Predefined Asset Metadata Removed?

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#17168] [Q&A / Support] [answered]
Was "Predefined Asset Metadata" removed in recent versions of Pimcore (11.x)? I see it on 10.6:
<img width="266" alt="image" src="https://github.com/pimcore/pimcore/assets/51059/b0cddecb-73ad-43b5-b9dc-48f430361f72">
but not 11.2:
<img width="314" alt="image" src="https://github.com/pimcore/pimcore/assets/51059/de16d9b4-989c-471f-ac31-f6d9d030e742">
Or did it morph into "Asset Metadata Class Definitions"?

**Comment by fashxp** [comment, marked as answer]
Predefined Asset Metadata is community edition, Asset Metadata Class Definitions is an enterprise edition drop-in.

**Reply by cancan101** [reply]
If I am on community edition using "Predefined Asset Metadata" and I upgrade to Enterprise, will my configurations be converted to "Class Definitions"?

Either way, documentation on these two different Metadata types (and their differences) would be good as I could not find any guidance.

**Reply by fashxp** [reply]
no, there is no conversion possible, as the asset metadata class definition offer much more options.

**Comment by kingjia90** [comment]
As soon as one installs the "Asset Metadata Class definitions" bundle, the "Predefined Asset Metadata" is hidden by
https://github.com/pimcore/asset-metadata-class-definitions/blob/4c7e45a5f51616508885b09a7a1cbfcaca4deba5/src/Resources/public/js/pimcore/metadataStartup.js#L76

**Reply by cancan101** [reply]
FYI, that link looks to be to a private repos.
